src/plugin-store.svelte

```diff
@@ -74,8 +74,20 @@ $: editorPlugins = filteredPlugins.filter((it) => it.kind === "editor");
 $: menuPlugins = filteredPlugins.filter((it) => it.kind === "menu");
 $: validatorPlugins = filteredPlugins.filter((it) => it.kind === "validator");
 
+//#region UI
+
 let pluginStore: Element;
 
+function openPluginDownloadUI() {
+    document.dispatchEvent(new Event("open-plugin-download"));
```

Comment:
Wouldn't we dispatch this event too high on the DOM tree?
we would need to dispatch it on some element this component